Dataset 1

In [1]:
import matplotlib.pyplot as plt
import pandas as pd
import numpy as np
from sklearn.model_selection import train_test_split
from sklearn.pipeline import make_pipeline
from sklearn.preprocessing import StandardScaler
from sklearn.linear_model import LinearRegression
from sklearn.linear_model import LogisticRegression

In [11]:
df =pd.read_csv('/content/car.data', header = None)
df

0      1      2     3      4     5      6
0     vhigh  vhigh      2     2  small   low  unacc
1     vhigh  vhigh      2     2  small   med  unacc
2     vhigh  vhigh      2     2  small  high  unacc
3     vhigh  vhigh      2     2    med   low  unacc
4     vhigh  vhigh      2     2    med   med  unacc
...     ...    ...    ...   ...    ...   ...    ...
1723    low    low  5more  more    med   med   good
1724    low    low  5more  more    med  high  vgood
1725    low    low  5more  more    big   low  unacc
1726    low    low  5more  more    big   med   good
1727    low    low  5more  more    big  high  vgood

[1728 rows x 7 columns]

1) Source of data

https://archive.ics.uci.edu/ml/machine-learning-databases/car/
2) Brief description of data


   Car Evaluation Database was derived from a simple hierarchical
   decision model originally developed for the demonstration of DEX
   (M. Bohanec, V. Rajkovic: Expert system for decision
   making. Sistemica 1(1), pp. 145-157, 1990.). The model evaluates

3) What is the target? 
Price

4) Is this a classification or regression problem?
Regression 
5) How many features?
Attributes: buying, maint, doors, persons, lug_boot, safety.
6) How many rows of data.
1728 rows × 7 columns
7) What, if any, challenges do your foresee in cleaning, exploring, or modeling with this dataset?
Exploring and Modeling

Dataset 2

In [13]:
df =pd.read_csv('/content/sobar-72.csv')
df

behavior_sexualRisk  behavior_eating  behavior_personalHygine  \
0                    10               13                       12   
1                    10               11                       11   
2                    10               15                        3   
3                    10               11                       10   
4                     8               11                        7   
..                  ...              ...                      ...   
67                   10               14                       14   
68                   10               12                       15   
69                   10                8                       11   
70                    9               12                       13   
71                   10               14                       14   

    intention_aggregation  intention_commitment  attitude_consistency  \
0                       4                     7                     9   
1                      10                    14                     7   
2                       2                    14                     8   
3                      10                    15                     7   
4                       8                    10                     7   
..                    ...                   ...                   ...   
67                     10                    15                     6   
68                     10                    15                     8   
69                      6                    10                     6   
70                     10                    13                     6   
71                      6                    12                     7   

    attitude_spontaneity  norm_significantPerson  norm_fulfillment  \
0                     10                       1                 8   
1                      7                       5                 5   
2                     10                       1                 4   
3                      7                       1                 5   
4                      8                       1                 5   
..                   ...                     ...               ...   
67                     7                       5                15   
68                     8                       5                15   
69                     4                       3                13   
70                     6                       5                14   
71                     8                       5                15   

    perception_vulnerability  perception_severity  motivation_strength  \
0                          7                    3                   14   
1                          4                    2                   15   
2                          7                    2                    7   
3                          4                    2                   15   
4                          3                    2                   15   
..                       ...                  ...                  ...   
67                        14                   10                   15   
68                        14                    8                   12   
69                         9                    8                   14   
70                        13                   10                   13   
71                        12                   10                   10   

    motivation_willingness  socialSupport_emotionality  \
0                        8                           5   
1                       13                           7   
2                        3                           3   
3                       13                           7   
4                        5                           3   
..                     ...                         ...   
67                      13                           9   
68                      14                          11   
69                      12                    

1) Source of data
https://archive.ics.uci.edu/ml/machine-learning-databases/00537/

2) Brief description of data

Cervical Cancer Behavior Risk Data Set

3) What is the target?
ca_cervix (this is class attribute, 1=has cervical cancer, 0=no cervical cancer)
4) Is this a classification or regression problem?
Classification
5) How many features?
18 attributes
6) How many rows of data.
72 rows × 20 columns
7) What, if any, challenges do your foresee in cleaning, exploring, or modeling with this dataset?
Exploring and Modeling